docs/book/src/user/quick-start.md

```diff
@@ -407,19 +407,34 @@ kubeadmconfigtemplate.bootstrap.cluster.x-k8s.io/capi-quickstart-md-0 created
 
 #### Accessing the workload cluster
 
-To verify the control plane is up, check if the control plane is ready.
+The cluster will now start provisioning. You check status with:
+
+```bash
+kubectl get cluster --all-namespaces
 ```
+
+To verify the first control plane is up:
+
+```bash
 kubectl get kubeadmcontrolplane --all-namespaces
 ```
 
-If the control plane is deployed using a control plane provider, such as
-[KubeadmControlPlane] ensure the control plane is
-ready.
-```
-kubectl get clusters --output jsonpath="{range .items[*]} [{.metadata.name} {.status.ControlPlaneInitialized} {.status.ControlPlaneReady}] {end}"
```

Comment:
removed this because it wasn't showing any output for me, even after installing the CNI I get:
` [capi-quickstart  ] %`